web/src/engine/websites/iqiyi.ts

```diff
@@ -0,0 +1,46 @@
+import { Tags } from '../Tags';
+import icon from './iqiyi.webp';
+import { Chapter, DecoratableMangaScraper, type Manga } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import * as MH from './decorators/MH';
+import { FetchJSON, FetchRequest } from '../FetchProvider';
+
+type APIChapters = {
+    data: {
+        episodes: {
+            episodeId: number,
+            episodeOrder: number,
+            episodeTitle : string
+        }[]
+    }
+
+}
+function MangaExtractor(anchor: HTMLAnchorElement) {
+    const id = anchor.pathname;
+    const title = anchor.nextElementSibling.textContent.trim();
+    return { id, title };
+}
+
+@Common.MangaCSS(/^{origin}\/manhua\/detail_\S+\.html$/, '.detail-tit h1')
```

Comment:
The `\S+` term allows non-alphanumeric characters such as `/` or `-` is this by intention?

Reply:
Ah missed this one.